tests/results/TextRangeTest.py

```diff
@@ -7,10 +7,12 @@
 class TextRangeTest(unittest.TestCase):
 
     def test_fail_instantation(self):
-        with self.assertRaises(ValueError):
+        error_message = "End position can't be less than start position"
```

Comment:
Please put this string in a class variable called `END_LT_START_RE` , and then you can access it in this method, and in method `test_from_values`.